ocm-api-load.example.yaml

```diff
@@ -19,6 +19,9 @@ tests:
   register-new-cluster:
     freq: 1000
     per: "h"
+  register-existing-cluster:
```

Comment:
Oops; I overlooked this file in my previous PR and had only added it to the readme. Thank you!

Reply:
Don't worry, I didn't notice also